### PR TITLE
Added voter count to posts table

### LINKED ISSUE #17: Include voter count in post

### PR DESCRIPTION
**Pull request by shanev**
Fixes #17

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,7 +19,7 @@ build-linux: pkger
 build-docker:
 	docker build -t publicawesome/stakewatcher .
 
-migrate: pkger
+run-migrate: pkger
```
Comment (JakeHartnell) [inline Makefile:22]: Any chance this change might have an effect on one our scripts?

**Review by JakeHartnell — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (shanev) [reply Makefile:22]: Good point, let me check...

Comment (jhernandezb) [reply Makefile:22]: It doesn't affect, is just to locally test with automigrations

**Review by jhernandezb — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (shanev) [reply Makefile:22]: Looks like we are using `--auto-migrate` in scripts instead of this make target

**Pull request merged**